doc/source/advanced/executor.rst

```diff
@@ -22,20 +32,20 @@ To do this, users first need to specify a connection to their SQL database. This
 .. code-block:: python
 
 	import psycopg2
-	connection = psycopg2.connect("dbname=example_database user=example_user, password=example_password")
+	connection = psycopg2.connect("dbname=postgres_db_name user=example_user password=example_user_password")
 
```

Comment:
Should we also give an example here for sqlalchemy and say that it also works with sqlalchemy? Users are also responsible for installing psycopg2 or sqlalchemy on their own.

Reply:
I think that would make sense, can add a quick setup of the sqlalchemy engine just to give an example to users.